extensions/objects/templates/ImageWithRegionsEditor.js

```diff
@@ -172,7 +172,7 @@ oppia.directive('imageWithRegionsEditor', [
             if (newVal !== '') {
               // Loads the image in hanging <img> tag so as to get the
               // width and height.
-              $('<img/>').attr('src', $scope.getPreviewUrl(newVal)).on(
+              $('<img/>').attr('src', $scope.getPreviewUrl(newVal.name)).on(
                 'load', function() {
                   $scope.originalImageWidth = this.width;
                   $scope.originalImageHeight = this.height;
```

Comment:
This place we can later replace with getting the width and height from our filepath object, am I right?

Reply:
Yeah, we can. But I don't think there is need for that because the image tag created loads the image, so using this.width wou;d be fine too.